netdata-installer.sh

```diff
@@ -145,7 +150,7 @@ printf "\n# " >> netdata-installer.log
 date >> netdata-installer.log
 printf 'CFLAGS="%s" ' "${CFLAGS}" >> netdata-installer.log
 printf 'LDFLAGS="%s" ' "${LDFLAGS}" >> netdata-installer.log
-printf "%q " "${PROGRAM}" "${@}" >> netdata-installer.log
+printf "%s" "${PROGRAM}" "${@}" >> netdata-installer.log
 printf "\n" >> netdata-installer.log
```

Comment:
It would be nice to just fix the shellcheck warning being suppressed for the above lines, but it’s not critical for this PR.